sqlglot/transforms.py

```diff
@@ -296,19 +296,41 @@ def unqualify_unnest(expression: exp.Expression) -> exp.Expression:
     return expression
 
 
-def unnest_to_explode(expression: exp.Expression) -> exp.Expression:
+def unnest_to_explode(
+    expression: exp.Expression,
+    unnest_using_arrays_zip: bool = True,
+    support_unnest_func: bool = False,
+) -> exp.Expression:
     """Convert cross join unnest into lateral view explode."""
+
+    def handle_unnest(u: exp.Unnest, _has_multi_expr: bool) -> None:
+        if unnest_using_arrays_zip and _has_multi_expr:
+            # Use INLINE(ARRAYS_ZIP(...)) for multiple expressions
+            u.set(
+                "expressions",
+                [exp.Anonymous(this="ARRAYS_ZIP", expressions=u.expressions)],
+            )
```

Comment:
The function name could be more descriptive e.g. `def _unnest_zip_exprs(...)`

Reply:
Good idea, _unnest_zip_exprs is more readable